datasets/eu/cor_members/crawler.py

```diff
@@ -2,7 +2,7 @@
 
 from zavod import Context
 from zavod import helpers as h
-from zavod.helpers.positions import OccupancyStatus
+from zavod.logic.pep import OccupancyStatus
```

Comment:
Let's just double-check what the output here looks like after these changes before merging

Reply:
Still two conflicting statuses, but uses value now instead of __repr__ so it's a pareto improvement.

![Screenshot_2023-09-08_08-18-34](https://github.com/opensanctions/opensanctions/assets/235801/2b997d40-f8e8-4e4c-8e9d-313cc5841abb)